cf-custom-resources/lib/dns-cert-validator.js

```diff
@@ -85,36 +86,33 @@ let report = function (
  * Lastly, the function exits until the certificate is validated.
  *
  * @param {string} requestId the CloudFormation request ID
- * @param {string} appName the name of the application
- * @param {string} envName the name of the environment
- * @param {string} domainName the Common Name (CN) field for the requested certificate
- * @param {string} subjectAlternativeNames additional FQDNs to be included in the
- * Subject Alternative Name extension of the requested certificate
+ * @param {string} appName the application name
+ * @param {string} envName the environment name
+ * @param {string} certDomain the domain of the certificate
+ * @param {string} aliases the custom domain aliases
  * @param {string} envHostedZoneId the environment Route53 Hosted Zone ID
  * @param {string} rootDnsRole the IAM role ARN that can manage domainName
- * @param {string} isAliasEnabled whether alias is enabled
  * @returns {string} Validated certificate ARN
  */
 const requestCertificate = async function (
   requestId,
   appName,
   envName,
-  domainName,
-  subjectAlternativeNames,
+  certDomain,
+  aliases,
   envHostedZoneId,
   rootDnsRole,
-  isAliasEnabled,
   region
 ) {
   const crypto = require("crypto");
   const [acm, envRoute53, appRoute53] = clients(region, rootDnsRole);
-  var sansToUse =
-    isAliasEnabled === "false"
-      ? [`*.${envName}.${appName}.${domainName}`]
-      : subjectAlternativeNames;
+  const sansToUse = [`*.${certDomain}`];
```

Comment:
Do we initialize to this value to maintain backwards compatibility? (instead of leaving it empty) 

If so can we add a comment here 🙏